static/css/restyle/restyle.less

```diff
@@ -1602,6 +1602,13 @@ h3.author .transfer-ownership {
   h4 {
     clear: both;
     font-size: 12px;
+
+    .author-names{
+      text-overflow: ellipsis;
+      overflow:hidden;
```

Comment:
Can you adapt the style we're using all over the file and add a space between the colon and the first character, e.g `overflow: hidden`?